src/project/view/exportdialogmodel.cpp

```diff
@@ -46,6 +46,10 @@ ExportDialogModel::ExportDialogModel(QObject* parent)
                                      qtrc("project/export", "Compressed") + " (*.mxl)",
                                      qtrc("project/export", "Compressed MusicXML files"),
                                      "MusicXmlSettingsPage.qml"),
+        ExportType::makeWithSuffixes({ "mxml" },
+                                     qtrc("project/export", "Compressed") + " (*.mxml)",
+                                     qtrc("project/export", "Compressed MusicXML files"),
+                                     "MusicXmlSettingsPage.qml"),
```

Comment:
This is what makes the export  of .mxml possible via the GUI

Reply:
OK, well we _definitely_ don't want this. I thought you were only adding the ability to **read** the files. I should have looked more closely and probably not merged this late at night.

After several heated debates, we eventually justified keeping the `.xml` extension because it _used to be_ the standard for uncompressed MusicXML files and there are still some legacy programs that don't work with `.musicxml`.

I don't think a similar argument can be made for `.mxml`. As far as I can tell:
1. `.mxml` has never been the standard way to name a compressed MusicXML file.
2. No program besides MuseScore would ever write in this format, and MuseScore only did it on the command line.
3. Not even MuseScore was capable of reading in this format.
4. Nobody has ever had a problem with this.

Please correct me if I'm wrong on any of those points, in which case we can keep the ability to write them on the command line and read them in the GUI, but I think we should disable writing them in the GUI.

However, if the above points are correct then I think the appropriate course of action is to revert this PR and remove the ability to write `.mxml` files even on the command line.

Reply:
(Not revert the whole PR, but just undo these four added lines, and if we want to disable it on command line too, make some additional other changes)

Reply:
@cbjeukendrup, the deduplication of `.xml` and `.musicxml` in [MacOSXBundleInfo.plist.in](https://github.com/musescore/MuseScore/pull/13122/files#diff-0cd9cdf851942abf3e6eb1b1286845898eaaf43c8e99a921a82f7cd66498ced2) can stay, but what else from this PR would we want to keep if we remove the ability to read and write these files?

Reply:
Oh, I didn't get that we also want to remove the ability to _read_.

Reply:
We'd probably just want ro remove, or comment out or `#ifdef` out that section above.
To be honest: I just realized now that this is responssible for showing .mxml on the GUI, before I just went all the places where .mxl is mentioned and adjusted them

Reply:
We'd probably just want ro remove, or comment out or `#ifdef` out that section above. That'd make the option vanish for export/writing from the GUI, but keeps the option to write from the command line and keeps the option to read these files, from command line and GUI.

To be honest: I just realized now that this is responssible for showing .mxml on the GUI, before I just went all the places where .mxl is mentioned and adjusted them. Enabile write from the GUI was not what I inteded.

Reply:
See #13146

Reply:
The thing is, we don't want people to look at our program (GUI or code) and think that `.mxml` is a standard when it's not. If several other programs use `.mxml` then we could justify reading but not writing in that format. If it's just us using `.mxml` then I think we should ditch it and return to the standard behaviour.